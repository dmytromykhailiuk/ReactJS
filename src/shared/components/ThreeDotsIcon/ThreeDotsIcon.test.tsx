import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import { ThreeDotsIcon } from '.';

describe('ThreeDotsIcon', () => {
  it('should match first snepshot', () => {
    const wrapper = shallow(<ThreeDotsIcon />);
    expect(toJson(wrapper)).toMatchSnapshot('first');
  });
});
