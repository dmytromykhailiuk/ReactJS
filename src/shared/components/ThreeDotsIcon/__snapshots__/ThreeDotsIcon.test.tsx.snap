// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThreeDotsIcon should match first snepshot: first 1`] = `
<div
  className="icon"
>
  <svg
    className="bi bi-three-dots-vertical"
    fill="currentColor"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9.5 13a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z"
    />
  </svg>
</div>
`;
